pkg/services/store/validate.go

```diff
@@ -95,6 +95,11 @@ func (s *standardStorageService) validateUploadRequest(ctx context.Context, user
 		return success()
 	case EntityTypeImage:
 		return s.validateImage(ctx, user, req)
+	case EntityTypeArchive:
```

Comment:
it seems like entity types are a mix of:
* specific file types like `json`
* generic file types like `archive` and `image`
* schema types like `dashboard` which are also stored as JSON

could this cause issues in the future if we need to know what the archive's contains (search index backup or something else), or the specific file type of the archive (zip, tar, gzipped tar)?

Reply:
That's a good question, and this was one of the reasons I initially wanted to use `EntityTypeBlob` for the task. @ArturWierzbicki what do you think?

Reply:
in general, we are going to end up with a split between file kinds/types that are "raw", and those which have defined schemas. 

I think both the `archive` and `image` generic types are OK - they are a useful, high-level descriptions of files with different validation needs. I guess the question is whether we should have them at all, OR should we just have `Blob` (+ schema'd types), and rely on the file extension & mime type detection. Thoughts?

Reply:
👍 seems like we still have time to think about this and adjust it later if needed